data_conversion/index.html

```diff
@@ -129,6 +131,19 @@ <h1>Stellar Mass-Halo Mass Relation</h1>
         </p>
     </div>
 
+    <div class="plotrow" id="sfr_halo">
```

Comment:
Would it not be better to include this at the `Star Formation Rates` section?

Reply:
I wanted to keep it separate as this is directly not observable and has no direct data comparison.